cassiopeia/preprocess/UMI_utils.py

```diff
@@ -659,61 +659,40 @@ def correct_umis_in_group(
     corrected and merged rather arbitrarily. We are looking into this.
 
     Args:
-        input_df: Input DataFrame of alignments.
+        cell_group: Input DataFrame of alignments.
         max_umi_distance: Maximum Hamming distance between UMIs
             for error correction.
 
     Returns:
-        A DataFrame of error corrected UMIs within the grouping.
-    """
+        A DataFrame of error corrected UMIs within the grouping, and the
+        number of corrected UMIs
```

Comment:
Need an extra indentation in any line after the first in `Returns` statements. Here and elsewhere.

Reply:
Oh man I keep forgetting to do this. My bad.